azure-pipelines/templates/steps/vsts-release.yml

```diff
@@ -0,0 +1,79 @@
+# File: azure-pipelines\templates\steps\nightly-snapshot-releases\vsts-release.yml
+# Description: Template to assemble and publish Snapshot/Latest versions of ${{ parameters.projectName }}
+# Variable: 'ENV_VSTS_MVN_ANDROIDADACCOUNTS_USERNAME' was defined in the Variables tab
+# Variable: 'mvnAccessToken' was defined in the Variables tab
+# Variable: 'folderName' refers to the project folder
+# Variable: 'gradleTasks' list all the gradle tasks to be used.
+# Variable: 'accessTokenName'  secret used to access the private azure devops maven feed.
+# Variable: 'runUnitTest' boolean: refers to whether the pipeline will run unit tests or not.
+# https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate
+# Variable: 'versionType' type of version (Latest, Snapshot)
+
+parameters:
+- name: versionType
+  default: ''
+  values:
+  - Snapshot
+  - Latest
+- name: projectName
+  default: none
+- name: gradleTasks
```

Comment:
I believe this is really `assembleGradleTask`?

Reply:
The commands passed here depend on the library invoking this e.g. projectName: clean projectName:assemble<VARIANT>